typebackend/views.py

```diff
@@ -18,6 +18,18 @@ def get(self,request):
         }
         return Response(success)
 
+class UserNameAvailability(APIView):
+    def get(self,request):
+        username=request.GET.get('username',None)
+        
+        data={
+            'is_taken':User.objects.filter(username=username).exists(),
```

Comment:
Does `exists` return a bool? or the user object?

Reply:
Boolean